contributors/design-proposals/access.md

```diff
@@ -37,7 +37,7 @@ application or misconfiguration of K8s.
   - K8s API Path - attack by sending network messages to any K8s API endpoint.
   - Insider Path - attack on K8s system components. Attacker may have
 privileged access to networks, machines or K8s software and data. Software
-errors in K8s system components and administrator error are some types of threat
+errors in K8s system components and administrator errors are some types of threat
```

Comment:
"administrator error" is appropriate (c.f. ["human error"](http://www.dictionary.com/browse/human-error))